### PR TITLE
Paginate products

### PR DESCRIPTION
**Pull request by carlgoshert**

### CODE REVIEW

**Comment by carlgoshert** [comment]
[Products view pagination](https://trello.com/c/0IKhrHSI/50-products-view-pagination)

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -15,11 +15,7 @@ class ProductController extends Controller
      */
     public function index()
     {
-        if(\Auth::user()->is_admin == 1){
-          return view("products", ["products" => Product::all()]);
-        }else{
-          return redirect("/");
-        }
+      //
```
Comment (circuitfox) [inline app/Http/Controllers/ProductController.php]: Can you justify removing this? It was meant as a complement to `show`

File: database/seeds/DashboardTestSeeder.php [hunk 2]
```diff
@@ -11,8 +11,8 @@ class DashboardTestSeeder extends Seeder
      */
     public function run()
     {
-        $user = factory(\App\User::class)->create();
-        $orders = factory(\App\Order::class, 5)->create()->each(function ($o) use ($user) {
+      $user = factory(\App\User::class)->create();
+      $orders = factory(\App\Order::class, 5)->create()->each(function ($o) use ($user) {
             $o->user()->associate($user);
```
Comment (circuitfox) [inline database/seeds/DashboardTestSeeder.php]: Fix the indents here; this is noise that really isn't necessary.

**Review by circuitfox — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/ProductController.php (hunk 1 above)
Comment (carlgoshert) [reply app/Http/Controllers/ProductController.php]: It is never used unless you type '/products' into the URL

**Review by circuitfox — APPROVED** [review]

**Pull request merged**